packages/image-block/src/settings.ts

```diff
@@ -1,28 +1,299 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import { AssetChooserObjectType, AssetInputSize, defineSettings } from '@frontify/guideline-blocks-settings';
+import {
+    AssetChooserObjectType,
+    AssetInputSize,
+    DropdownSize,
+    IconEnum,
+    MultiInputLayout,
+    NotificationStyleType,
+    appendUnit,
+    defineSettings,
+    numericalOrPixelRule,
+    presetCustomValue,
+} from '@frontify/guideline-blocks-settings';
+
+import {
+    Alignment,
+    BorderStyle,
+    CaptionPosition,
+    ImageSecurity,
+    Padding,
+    Ratio,
+    paddingValues,
+    radiusValues,
+} from './types';
+
+const POSITIONING_ID = 'positioning';
+const HAS_BACKGROUND_ID = 'hasBackground';
+const HAS_BORDER_ID = 'hasBorder';
+const PADDING_CHOICE_ID = 'paddingChoice';
+const PADDING_CUSTOM_ID = 'paddingCustom';
+const RADIUS_CHOICE_ID = 'radiusChoice';
+const RADIUS_CUSTOM_ID = 'radiusCustom';
 
 export const settings = defineSettings({
     basics: [
         {
-            id: 'image',
-            type: 'assetInput',
-            size: AssetInputSize.Small,
-            objectTypes: [AssetChooserObjectType.ImageVideo],
+            id: 'imageSection',
+            type: 'sectionHeading',
+            label: 'Image',
```

Comment:
For me the label in the design is the same as for the link. Therefore, I would move the label to the id `image` on line 44 and remove it here. Wdyt?

Reply:
Changed 👍🏻